samples/google-tag-manager.html

```diff
@@ -12,7 +12,7 @@
 </head>
 <body ng-app="sample">
 	<!-- Google Tag Manager, replace GTM-XXXXX with your container ID -->
-<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-53SG4H"
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-XXXXX"
```

Comment:
To be consistent with the other "GTM-XXXXX" [below] in the code ! (https://github.com/luisfarzati/angulartics/blob/master/samples/google-tag-manager.html#L21)

But, yeah, there is indeed 6 chars in the GTM id, not 5